.typo-ci.yml

```diff
@@ -1,17 +0,0 @@
-dictionaries:
```

Comment:
Just curiosity, are we just removing it or is it replaced by something else?

Reply:
this file was not being used by any pipeline... so removed it. Is it part of any other workflow i am not aware of?

Reply:
I think I had some report of this in the past, but not sure where/how. Anyways, this is the configuration for [typo CI](https://typoci.com/documentation) and we don't have it configure, so it can be removed